node/lib/Logger.ts

```diff
@@ -20,62 +22,89 @@ export class Logger implements ILogger {
             this.globalLogLevel = logLevel;
         }
 
-        let consoleTransport = new winston.transports.Console({
+        this.consoleTransport = new winston.transports.Console({
             colorize: true,
             level: this.globalLogLevel,
         });
 
         this.logger.configure({
             transports: [
-                consoleTransport,
+                this.consoleTransport,
             ],
         });
 
-        this.transportList.push(consoleTransport);
+        this.transportList.push(this.consoleTransport);
 
-        if (Logger.flag === true && filename ) {
-            let fileTransport = new winston.transports.File({
+        if (Logger.flag === true && filename) {
+            this.fileTransport = new winston.transports.File({
                 filename: filename,
                 handleExceptions: true,
                 level: this.globalLogLevel,
             });
 
             this.logger.configure({
```

Comment:
wouldn't it be cleaner to configure the logger with this.transportList once rather than inside these two ifs?